test/unit/core/algorithm/detail/blocking_algorithm_executor_test.cpp

```diff
@@ -85,14 +83,14 @@ TYPED_TEST(blocking_algorithm_executor_test, is_eof)
                                                    size_t,
                                                    TypeParam>;
 
-    alignment_executor_t exec{this->sequence_pairs, algorithm_t{dummy_alignment{}}};
+    alignment_executor_t exec{this->sequence_pairs, algorithm_t{dummy_algorithm{}}};
```

Comment:
I don't know if thats necessary: Should the `alignment_executor_t` template typename be renamed to something like `algorithm_executor_t` ?